docs/guide/mkdocs.yml

```diff
@@ -42,6 +42,7 @@ nav:
       - Open OnDemand: development_guide/ood.md
       - Dataverse Integration: development_guide/dataverse_integration.md
       - Contributing a Change: development_guide/contributing.md
+      - E2E Tests: development_guide/e2e_tests.md
       - GitHub Actions: development_guide/github_actions.md
```

Comment:
We may want to document the new GitHub action in the Guide section `development_guide/github_actions.md`